test/runtests.jl

```diff
@@ -18,7 +18,6 @@ for meth in filter(f -> isa(eval(f), Function), names(NLPModels))
   @test_throws(NotImplementedError, meth(model))
 end
 
-include("genrose.jl")
```

Comment:
That makes much more sense. I think those are testing only if the appropriate exception is being thrown, but testing specific code is better.
Can I do it in another PR?